k8s/migration/api/v1alpha1/vmwaremachine_types.go

```diff
@@ -20,16 +20,42 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type RDMDiskInfo struct {
+	// DiskName is the name of the disk
+	DiskName string `json:"diskName,omitempty"`
+	// DiskSize is the size of the disk in GB
+	DiskSize int64 `json:"diskSize,omitempty"`
+	// UUID is the unique identifier of the disk
+	UUID string `json:"uuid,omitempty"`
+	// DisplayName is the display name of the disk
+	DisplayName string `json:"displayName,omitempty"`
+	// OperationalState is the operational state of the disk
+	OperationalState []string `json:"operationalState,omitempty"`
+	// CinderBackendPool is the cinder backend pool of the disk
+	CinderBackendPool string `json:"cinderBackendPool,omitempty"`
+	// VolumeType is the volume type of the disk
+	VolumeType string `json:"volumeType,omitempty"`
+	// AvailabilityZone is the availability zone of the disk
+	AvailabilityZone string `json:"availabilityZone,omitempty"`
+	// Bootable indicates if the disk is bootable
+	Bootable bool `json:"bootable,omitempty"`
+	// DiskMode is the mode of the disk
+	Description string `json:"description,omitempty"`
+}
+
 type VMInfo struct {
-	Name       string   `json:"name"`
-	Datastores []string `json:"datastores,omitempty"`
-	Disks      []string `json:"disks,omitempty"`
-	Networks   []string `json:"networks,omitempty"`
-	IPAddress  string   `json:"ipAddress,omitempty"`
-	VMState    string   `json:"vmState,omitempty"`
-	OSType     string   `json:"osType,omitempty"`
-	CPU        int      `json:"cpu,omitempty"`
-	Memory     int      `json:"memory,omitempty"`
+	Name             string        `json:"name"`
+	Datastores       []string      `json:"datastores,omitempty"`
+	Disks            []string      `json:"disks,omitempty"`
+	Networks         []string      `json:"networks,omitempty"`
+	IPAddress        string        `json:"ipAddress,omitempty"`
+	VMState          string        `json:"vmState,omitempty"`
+	OSType           string        `json:"osType,omitempty"`
+	CPU              int           `json:"cpu,omitempty"`
+	Memory           int           `json:"memory,omitempty"`
+	Annotation       string        `json:"annotation,omitempty"`
+	CustomAttributes []string      `json:"customAttributes,omitempty"`
```

Comment:
Discussion: do we need to store these in VMwareMachine? Or we just parse out what we need and drop these two?